.github/workflows/build.yml

```diff
@@ -25,3 +25,45 @@ jobs:
 
       - name: Run build
         run: yarn build
+
+  # Publishing is done in a separate job to allow
+  # for all matrix builds to complete.
+  release:
+    needs: test
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    strategy:
+      fail-fast: false
+
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: Check out repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      # Fetch tags and describe the commit before the merge commit
+      # to see if it's a version publish
+      - name: Fetch tags
+        run: |
+          git fetch --tags
+          if git describe --exact-match --match "v*.*.*" HEAD^2
+          then
+            echo "Found version commit tag. Publishing."
+            echo "::set-env name=publish::true"
+          else
+            echo "Version commit tag not found. Not publishing."
+          fi
+
+      - name: Publish
+        if: env.publish == 'true'
+        env:
+          NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
If we decide to use this, we'll need add a secret cc @aiwilliams

Reply:
> Secrets are environment variables that are encrypted and only exposed to selected actions. Anyone with collaborator access to this repository can use these secrets in a workflow.

I suppose we shouldn't add collaborators we don't trust to build workflows 😬 

Okay, secret is installed.

Reply:
Oh, note to everyone: we'll need to add the secret to each repository that does this, just as we had to do with Travis.